src/status_im/contacts/events.cljs

```diff
@@ -264,26 +253,31 @@
   :load-default-contacts!
   [(inject-cofx ::get-default-contacts-and-groups)]
   (fn [{:keys [db default-contacts default-groups]} _]
-    (let [{:keys [groups] :contacts/keys [contacts]} db]
+    (let [{:contacts/keys [contacts] :group/keys [contact-groups]} db]
       {:dispatch-n (concat
-                     (prepare-default-groups-events groups default-groups)
+                     (prepare-default-groups-events contact-groups default-groups)
                      (prepare-default-contacts-events contacts default-contacts)
                      (prepare-add-chat-events contacts default-contacts)
                      (prepare-bot-commands-events contacts default-contacts)
                      (prepare-add-contacts-to-groups-events contacts default-contacts))})))
```

Comment:
can you elaborate? result of function can be empty , so it cant be `(concat [[item1]] [[item2]] nil [])` and result should be `[[item1] [item2]]`